### PR TITLE
Adding Flake8 config file 

### PR DESCRIPTION
**Pull request by RonTuretzky**
attempting to push changes according to what @bennypowers [linked](https://flake8.pycqa.org/en/latest/user/configuration.html) in an earlier PR

### CODE REVIEW

**Comment by RonTuretzky** [comment]
- [ ] Add a line that's longer then 120 
- [ ]  commit and push 
- [ ] check if ci indeed fails ( to make sure there's no false positive) 
- [ ] if ci fails,  revert commit 
- [ ] squash revert + configfile + cmd line change when merging

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```
Comment (bennypowers) [inline .flake8:2]: Reserving this comment to block merge until ci has been manually tested, see @RonTuretzky 's comment above

**Review by bennypowers — APPROVED** [review]

File: .flake8 (hunk 1 above)
Comment (kobihk) [inline .flake8:2]: @RonTuretzky please add a new line

**Review by kobihk — CHANGES_REQUESTED** [review]

File: .flake8 (hunk 1 above)
Comment (RonTuretzky) [reply .flake8:2]: @bennypowers Flake8 detected a line that was too long 

image

proceeding to next steps

Comment (RonTuretzky) [reply .flake8]: Implemented

**Comment by RonTuretzky** [comment]
@bennypowers squash + force push was completed as instructed

**Review by bennypowers — APPROVED** [review]

**Review by NicolePauzner — APPROVED** [review]
LGTM

**Review by EliasAndry — APPROVED** [review]

**Review by kobihk — APPROVED** [review]

**Pull request merged**